scripts/util.js

```diff
@@ -0,0 +1,41 @@
+const tmp = require("tmp");
```

Comment:
Don't create a file named `util` just for exporting a single function. The general name does not add anything, and this could easily turn out to be the junkyard of functions. A better name for this file at its current form is `createTypescriptProgram.js` with a single default export

Reply:
True, it started out as more then that but the more I worked the less it contained.
DOne